src/glcontrol/cfgtools/specs.py

```diff
@@ -0,0 +1,43 @@
+from enum import Enum
+
+from pydantic import BaseModel
+
+from glcontrol.cfgtools.base import Parseable
+
+
+class CameraSpec(BaseModel, Parseable):
+    name: str
+    input_type: str
+    id: dict
+    options: dict = {}
```

Comment:
To prevent the default `dictionary` / `list` bugs, you can use `Field(default_factory=...)` here and below.

```suggestion
    options: dict = Field(default_factory=dict)
```